controller/src/main/java/io/pravega/controller/server/eventProcessor/LocalController.java

```diff
@@ -466,10 +466,10 @@ public String retrieveDelegationToken() {
             }
         });
     }
-
+    
     @Override
```

Comment:
revert this class

Reply:
This class is used by Integration Tests when invoking Controller operations.
Hence added listKeyValueTables() here too. 
Integration tests need to be fixed to use ControllerImpl.java in client instead.